helm-chart/renku/templates/secrets-storage/pdb.yaml

```
@@ -0,0 +1,17 @@
+{{- if or (gt (int .Values.secretsStorage.replicaCount) 1) (and .Values.secretsStorage.autoscaling.enabled (gt (int .Values.secretsStorage.autoscaling.minReplicas) 1)) }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "renku.fullname" . }}-secrets-storage
+  labels:
+    app: renku-secrets-storage
+    chart: {{ template "renku.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  maxUnavailable: 50%
```

This may be like this in other places. But `maxUnavailable` will round up. So if you have 1 pod you will get fully disrupted. We should switch to `minAvailable` so that if we have 1 pod we will not get fully disrupted.

```suggestion
  minAvailable: 50%
```